packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -997,18 +953,10 @@ class HotRunner extends ResidentRunner {
       fullRestart: false,
       reason: reason,
       overallTimeInMs: reloadInMs,
-      finalLibraryCount: firstReloadDetails != null
-        ? firstReloadDetails['finalLibraryCount'] as int
-        : null,
-      syncedLibraryCount: firstReloadDetails != null
-        ? firstReloadDetails['receivedLibraryCount'] as int
-        : null,
-      syncedClassesCount: firstReloadDetails != null
-        ? firstReloadDetails['receivedClassesCount'] as int
-        : null,
-      syncedProceduresCount: firstReloadDetails != null
-        ? firstReloadDetails['receivedProceduresCount'] as int
-        : null,
+      finalLibraryCount: firstReloadDetails['finalLibraryCount'] as int,
```

Comment:
Does this one have to be null rather than 0?

Reply:
this is measuring the final library count in the VM, which I we don't know if the hot reload doesn't run. I could make it 0, but it would still be a sentinel value